src/locale/languages.ts

```diff
@@ -56,6 +57,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.th, name: 'ภาษาไทย – Thai'},
   {code2: AppLanguage.tr, name: 'Türkçe – Turkish'},
   {code2: AppLanguage.uk, name: 'Українська – Ukrainian'},
+  {code2: AppLanguage.vi, name: 'Tiếng Việt'},
```

Comment:
```suggestion
  {code2: AppLanguage.vi, name: 'Tiếng Việt – Vietnamese'},
```
Could I suggest adding the name in English too, to keep it in line with the other entries?

Reply:
Ah yes. I missed that somehow sorry. Amended the latest commit to keep it clean. Thanks!